.env.development

```diff
@@ -1,5 +1,5 @@
 KURL_INSTALLER_URL="https://staging.kurl.sh/installer"
-KURL_BUNDLE_URL="https://k8s.staging.kurl.sh/bundle"
+KURL_BUNDLE_URL="https://staging.kurl.sh/bundle"
```

Comment:
this is basically reverting the previous PR

Reply:
Got it, let the requests go to [k8s](https://k8s.kurl.sh/bundle)  instead of [Cloudflare]( https://kurl.sh/bundle), writing it here for my own memory)